prow/config/inrepoconfig.go

```diff
@@ -106,7 +107,15 @@ func prowYAMLGetter(
 		return nil, err
 	}
 
-	mergeMethod := c.Tide.MergeMethod(orgRepo)
+	// TODO(mpherman): This is to hopefully mittigate issue with gerrit merges. Need to come up with a solution that checks
+	// each CLs merge strategy as they can differ. ifNecessary is just the gerrit default
+	var mergeMethod github.PullRequestMergeType
+	if c.Gerrit.DeckURL != "" {
```

Comment:
I think `OrgReposConfig` is more reliable than DeckURL

Reply:
Done